docs/faq.rst

```diff
@@ -758,6 +764,13 @@ This has some pros and cons, though:
   machine (if you share same backup repo between multiple machines) or if
   your local chunks cache was lost somehow.
 
+Above workaround is necessary only on the run when cache is being recreated,
+and once it completes may be reverted, with borg cache behaving as previously.
+Which means that if you don't want this behaviour, it's enough to have it in place
+only on first run if the cache got removed, but you'll need to keep it there in case
+of multiple machines writing to same repo, since in that scenario cache will be
+constantly getting invalidated.
```

Comment:
and just remove that text again, i guess it is rather confusing than helping.

either one is wanting more speed and accepts high space usage.

or one does not, then one needs to apply the described hack at any time (and needs to keep it).

Reply:
Yeah, with the above explanation it's clear when it's created and what it's used for, so this is superfluous.